tensorboard/test_util.py

```diff
@@ -22,13 +22,20 @@
 from __future__ import division
 from __future__ import print_function
 
+import contextlib
+import functools
 import logging
+import os
+import sqlite3
 import threading
 
 import tensorflow as tf
 
+from tensorboard import db
 from tensorboard import util
 
+db.TESTING_MODE = True
```

Comment:
Performing any action when the module is loaded seems dangerous to me. What do you think about doing this in `tf.test.TestCase.setUp`?

Alternately, if there is a way to make Bazel enforce that this module can only be a dependency of a `py_test` rule (which should be sufficient to enforce that it's never called from prod code—right?) then I'd be happy.